.travis.yml

```diff
@@ -24,9 +24,6 @@ jobs:
 
       script:
       - travis-ci/setup_tasks.sh
-    - <<: *run_tests
-      env:
-        - IMAGE=centos:centos7 TASK=tests COMPONENTS=udt,myproxy,ssh
     - <<: *run_tests
       env:
         - IMAGE=centos:centos6 TASK=tests COMPONENTS=gram5
```

Comment:
What does that change do?

Reply:
Nevermind, I only checked this singular commit w/o looking at the complete changes.